src/app/main/component/shared/components/form-base/form-base.component.ts

```diff
@@ -84,7 +84,7 @@ export class FormBaseComponent implements ComponentCanDeactivate {
             } else {
               this.router.navigate([this.previousPath]);
             }
-          } else if (isUbsOrderSubmit) {
+          } else if (isUbsOrderSubmit && confirm === null) {
```

Comment:
Can you use !confirm?

Reply:
Here "null" plays the role of the third parameter in confirm and must be different from "false".